### ISSUE TITLE
Change to avoid specific XF versions

### ISSUE THREAD

**Issue by jdluzen** [tbaggett/xfgloss#6]
Thanks for this project, I just started looking into it!

Is it feasible to change the Nuget package to require a minimum version of XF? I know some packages are forced to require a specific version for compatibility reasons, and others can do simple minimums.

**Comment by tbaggett** [comment]
Yeah, XFGloss can be changed so that v2.3.1.114 is the minimum XF version instead of the required one. See my reasons for requiring a specific version below, but if you want to test XFGloss against XF beta/alpha releases, I'm okay with changing it to the minimum version. Be aware that I may switch it back though if my concerns prove to be valid. 😄 

I locked the version because some of the XFGloss enhancements interact with internal implementations in XF, so XF updates are more likely to break them. 

I'm also concerned about getting issues raised against XF beta/alpha releases, as I only use Xamarin stable channel releases for development.

Finally, I took a cue from XF itself, which requires v23.3.0 of the Android support libraries.
